core/src/main/kotlin/org/sourcegrade/jagr/core/export/submission/GradleSubmissionExporter.kt

```diff
@@ -58,6 +58,17 @@ class GradleSubmissionExporter @Inject constructor(
             name = graderJar?.info?.name ?: DEFAULT_EXPORT_NAME
         }
         writeSkeleton()
+        if (graderJar != null && graderJar.configuration.exportBuildScriptPath != null) {
+            addResource {
+                name = "build.gradle.kts"
+                val buildScriptResourceName = graderJar.configuration.exportBuildScriptPath!!
```

Comment:
Use `checkNotNull` instead of `!!`

Reply:
Actually, this can be simplified by saving the `exportBuildScriptPath` in its own val before the if statement